src/components/Person/Person.jsx

```diff
@@ -1 +1,51 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+  let partner = '';
+
+  if (isMarried && sex === 'm') {
+    partner = 'wife';
+  } else if (isMarried && sex === 'f') {
+    partner = 'husband';
+  } else {
+    partner = null;
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      )}
+
+      <p className="Person__partner">
+        {partner === null
+          ? `I am not married`
+          : `${partnerName} is my ${partner}`}
```

Comment:
It's better to put this logic into an additional variable. Also, you already have variable isMarried, you don't need to take care of partner === null, your else in your if-else statement isn't necessity, just because you already can check everything you need with isMarried